forge/src/lib.rs

```diff
@@ -24,9 +27,11 @@ pub trait TestFilter {
     fn matches_path(&self, path: impl AsRef<str>) -> bool;
 }
 
+// TODO: Why did we add this again?
+pub static CALLER: Lazy<Address> = Lazy::new(Address::random);
```

Comment:
i added this because when running forked tests against mainnet, you need to have a non-zero balance to deploy the live network, check `call_committing` in `executor/mod.rs` and `deploy` in `fuzz/mod.rs`. it might be unneeded?